dhis-2/dhis-api/src/main/java/org/hisp/dhis/scheduling/JobType.java

```diff
@@ -100,6 +100,7 @@
     TRACKER_IMPORT_JOB( false ),
     TRACKER_IMPORT_NOTIFICATION_JOB( false ),
     TRACKER_IMPORT_RULE_ENGINE_JOB( false ),
+    SQL_VIEW_UPDATE( true ),
```

Comment:
Just wondering, do we want to call this `MATERLAIZED_SQL_VIEW_UPDATE` just to make it clear we are only updating materialized views/materialized views is the only type where updating applies?

Reply:
renamed.